src/components/ScoreInput.js

```diff
@@ -10,6 +10,14 @@ const sliderValues = {
 };
 
 const ScoreInput = React.createClass({
+  update(e) {
+    if (e.target.classList.contains('score-picker')) {
+      this.updateScoreLabel(e);
+    }
+    const scoreVal = document.getElementById(this.props.scope + '_score').value;
+    const commentVal = document.getElementById(this.props.scope + '_comment').value;
```

Comment:
I think we can use `refs` instead of  `getElementById`.

On our input do:
`<input ref="myInput" />`
Then: 
```javascript
var input = this.refs.myInput;
var inputValue = input.value;
```

https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute